.gitignore

```diff
@@ -1 +1 @@
-node_modules
+zinnia
```

Comment:
Should we ignore `zinnia.exe` as well, to support developers on Windows?

```suggestion
zinnia
zinnia.exe
```

Reply:
Yes, we should build inclusive community :)